### ISSUE TITLE
NewLines lost in multi-line task attributes

### ISSUE THREAD

**Issue by kzu** [Microsoft/msbuild#1402]
This manifests itself in Message and WriteLinesToFiles but I guess it's a general issue.

Repro project:

```
<?xml version="1.0" encoding="utf-8"?>
<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">

	<Target Name="Build">
		<Message Importance="high" Text="%(None.Identity):
		  Value1=%(Value1)
		  Value2=%(Value2)" />

		<WriteLinesToFile File="newlinestring.txt"
						  Lines="%(None.Identity):
		  Value1=%(Value1)
		  Value2=%(Value2)" />
	</Target>

	<ItemGroup>
		<None Include="A">
			<Value1>A1</Value1>
			<Value2>A2</Value2>
		</None>
		<None Include="B">
			<Value1>B1</Value1>
			<Value2>B2</Value2>
		</None>
	</ItemGroup>
</Project>
```

MSBuild 15.0 output:

```
Build:
  A:     Value1=A1     Value2=A2
  B:     Value1=B1     Value2=B2
```

MSBuild 14.0 output:

```
Build:
  A:
                  Value1=A1
                  Value2=A2
  B:
                  Value1=B1
                  Value2=B2
```

Same thing on the written file.

**Comment by rainersigwald** [comment]
Looks like this is yet another xml-newline-normalization victim. It should be fixed on full-framework MSBuild by #1378.

```
Microsoft (R) Build Engine version 15.1.415.7918
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 11/28/2016 2:44:56 PM.
Project "S:\work\autoimports\kzu.proj" on node 1 (default targets).
Build:
  A:
                  Value1=A1
                  Value2=A2
  B:
                  Value1=B1
                  Value2=B2
Done Building Project "S:\work\autoimports\kzu.proj" (default targets).


Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:00.54
```

@AndyGerlicher this behavior might bump the priority of fixing the .NET Core on Windows implementation for #1340 . . . which we still don't know how to do :(

**Comment by rainersigwald** [comment]
Closing as a duplicate of #1401. Thanks for reporting, @kzu.
